life_expectancy/tests/test_load_save_data.py

```diff
@@ -0,0 +1,18 @@
+"""Tests for the load and save module"""
+from unittest.mock import patch
+import unittest.mock as mock # pylint: disable=consider-using-from-import
+import pandas as pd
+from life_expectancy.load_save_data import load_data, save_data
+from . import OUTPUT_DIR
+
+
+def test_load_data() -> None:
+    """test load_data is a panda dataframe"""
+    raw_data = load_data()
+    assert isinstance(raw_data, pd.DataFrame)
```

Comment:
This function should be patched so that it doesn't consume data from the OUTPUT_DIR, and instead it consumes a fixture from the FIXTURES_DIR